doc/dev_guide/building_from_source.rst

```diff
@@ -112,7 +112,7 @@ the preferences can differ. But strategically the steps are always the same.
                autoconf automake libtool \
                readline ncurses openssl libunwind icu \
                python27 py27-pip py27-setuptools py27-daemon \
-               py27-msgpack py27-yaml py27-argparse py27-six py27-gevent
+               py27-msgpack py27-yaml py27-six py27-gevent
```

Comment:
Do we really use Python2? @Totktonada

Reply:
As I can see, related issues are still open so I decided lo leave it as is (https://github.com/tarantool/doc/issues/1757)

Reply:
It is so for now. Should be resolved within two weeks I guess.